lib/src/features/create_properties/presentation/screens/create_properties.dart

```diff
@@ -1,42 +1,178 @@
+import 'package:chip_list/chip_list.dart';
```

Comment:
If you could consider refactoring by reusing widgets and extracting static views (constant widgets) into their separate files, it would make your main screen less cluttered and more efficient.